integration/use-cases/create-private-registry.js

```diff
@@ -2,53 +2,68 @@ const axios = require("axios");
 const utils = require("./lib/utils");
 
 test("Creates a private registry", async () => {
-  await page.goto(getUrl("/#/c/default/ns/default/config/repos"));
-
-  await expect(page).toFillForm("form", {
-    token: process.env.ADMIN_TOKEN,
+  var token = process.env.USE_MULTICLUSTER_OIDC_ENV ? undefined : process.env.ADMIN_TOKEN;
+  page.on("response", response => {
+    // retrieves the token after the oidc flow, note this require "--set-authorization-header=true" flag to be enabled in oauth2proxy
+    token = response.headers()["authorization"] || token;
   });
 
-  await page.evaluate(() =>
-    document.querySelector("#login-submit-button").click()
+  await utils.login(
+    page,
+    process.env.USE_MULTICLUSTER_OIDC_ENV,
+    "/#/c/default/ns/default/config/repos",
+    process.env.ADMIN_TOKEN,
+    "kubeapps-operator@example.com",
+    "password",
   );
 
+  // wait for the loading msg to disappear
+  await page.waitForFunction(() => !document.querySelector("cds-progress-circle"));
+
   await expect(page).toClick("cds-button", { text: "Add App Repository" });
 
   const randomNumber = Math.floor(Math.random() * Math.floor(100));
   const repoName = "my-repo-" + randomNumber;
-  await page.type("input[placeholder=\"example\"]", repoName);
+  await page.type('input[placeholder="example"]', repoName);
 
   await page.type(
-    "input[placeholder=\"https://charts.example.com/stable\"]",
-    "http://chartmuseum-chartmuseum.kubeapps:8080"
+    'input[placeholder="https://charts.example.com/stable"]',
+    "http://chartmuseum-chartmuseum.kubeapps:8080",
   );
 
   await expect(page).toClick("label", { text: "Basic Auth" });
 
   // Credentials from e2e-test.sh
-  await page.type("input[placeholder=\"Username\"]", "admin");
-  await page.type("input[placeholder=\"Password\"]", "password");
+  await page.type('input[placeholder="Username"]', "admin");
+  await page.type('input[placeholder="Password"]', "password");
 
   // Open form to create a new secret
   const secret = "my-repo-secret" + randomNumber;
+
   try {
     // TODO(andresmgot): Remove this line once 2.3 is released
     await expect(page).toClick("cds-button", { text: "Add new credentials" });
-  } catch(e) {
-    await expect(page).toClick(".btn-info-outline", { text: "Add new credentials" });
+  } catch (e) {
+    await expect(page).toClick(".btn-info-outline", {
+      text: "Add new credentials",
+    });
   }
```

Comment:
we can remove this try-catch now

Reply:
When testing in my fork I didn't build the new CI images (for saving time), and I would still need these lines. Now they can be safely removed, thanks for the reminder!

Reply:
No, I remove them indeed. However, the CI started failing since the selectors weren't been found, so I reverted the change.

https://app.circleci.com/pipelines/github/kubeapps/kubeapps/2614/workflows/dee98a52-3d20-47a6-9a76-a1fa16351eaf/jobs/42596/artifacts

```log
 FAIL  use-cases/operator-deployment.js (108.546 s)

 RUNS  ...
  ● Deploys an Operator

    TimeoutError: Element div.modal-dialog.modal-md > div > div.modal-body > div > div > cds-button:nth-child(2) (text: "Delete") not found

      waiting for function failed: timeout 30000ms exceeded
```

But, since the tests were failing because of other reasons, I'm gonna trigger again a new build and let's see what happens.
If it is still failing I guess it will be something related to the built image.